### PR TITLE
Add airy design with business focus + add documentation + bump to Twitter Bootstrap 5, Font Awesome 6

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: .env_acceptance.dist [hunk 1]
```diff
@@ -1,7 +0,0 @@
-###> symfony/framework-bundle ###
-APP_ENV=prod
-APP_DEBUG=0
-APP_SECRET=67d829bf61dc5f87a71fd814e2c9f629
-###< symfony/framework-bundle ###
-
-DEBUG_COOKIE=
```
Comment (samsonasik) [inline .env_acceptance.dist:7]: This file is needed for acceptance test show production env https://github.com/rectorphp/getrector.org/blob/558aca6b5b71a62d38955cc1eb5ef48fe1e30fdd/.github/workflows/acceptance_test.yaml#L25

File: package.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
     "name": "getrector-org",
     "dependencies": {
-        "@fortawesome/fontawesome-free": "^5.9.0",
-        "@fortawesome/free-brands-svg-icons": "^5.9.0",
-        "@fortawesome/free-regular-svg-icons": "^5.9.0",
-        "@fortawesome/free-solid-svg-icons": "^5.9.0",
+        "@fortawesome/fontawesome-free": "^6.2",
+        "@fortawesome/free-brands-svg-icons": "^6.2",
+        "@fortawesome/free-regular-svg-icons": "^6.2",
+        "@fortawesome/free-solid-svg-icons": "^6.2",
```
Comment (samsonasik) [inline package.json:7]: Current deploy script on getrector.org doesn't run 

```
node install && yarn dev
```

so probably require to update after deployed

**Comment by samsonasik** [comment]
Ref for why acceptance test is needed https://github.com/rectorphp/getrector.org/pull/723

File: .env_acceptance.dist (hunk 1 above)
Comment (TomasVotruba) [reply .env_acceptance.dist]: Fixed

File: package.json (hunk 2 above)
Comment (TomasVotruba) [reply package.json:7]: Could you add that line to the deploy script now? 

So we don't have to think about it in the future and have it covered :+1:

Comment (samsonasik) [reply package.json:7]: I can see `yarn build` in the script, is that same with `node install && yarn dev` ?

Comment (TomasVotruba) [reply package.json:7]: I'm not much expert on this one :) 

Locally I've run:

```bash
yarn upgrade # similar to "composer update"
yarn dev # to build the assets
```

Comment (samsonasik) [reply package.json:7]: It seems `npm install` actually needed, to download the updated packages first, I updated the script so it install the dependencies and build the webpack

Comment (samsonasik) [reply package.json:7]: me too, I currently apply: `npm install && yarn build`, yarn build seems for production so it minified result on generated webpack, let's see if that's working on next build

**Comment by TomasVotruba** [comment]
## How does the changes look like? :) 

1_demo_before
2_demo_after
3_home_before
4_home_after

**Comment by TomasVotruba** [comment]
Let's :ship: this, so we can iterate on it :)

**Comment by TomasVotruba** [comment]
It's on: https://getrector.org/ :partying_face:

**Pull request merged**